src/app/register/register.module.ts

```diff
@@ -0,0 +1,59 @@
+import { NgModule } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { RegisterRoutingModule } from './register-routing.module'
+import { RegisterComponent } from './pages/register/register.component'
+import { StepAComponent } from './components/step-a/step-a.component'
+import { StepBComponent } from './components/step-b/step-b.component'
+import { StepCComponent } from './components/step-c/step-c.component'
+import { FormPersonalComponent } from './components/form-personal/form-personal.component'
+import { FormPasswordComponent } from './components/form-password/form-password.component'
+import { FormNotificationsComponent } from './components/form-notifications/form-notifications.component'
+import { FormVisibilityComponent } from './components/form-visibility/form-visibility.component'
+import { FormTermsComponent } from './components/form-terms/form-terms.component'
+import { FormsModule, ReactiveFormsModule } from '@angular/forms'
+// tslint:disable-next-line: max-line-length
```

Comment:
@leomendoza123 I think we should avoid disabling tslint rules and put the code in different lines to avoid error rule max-line-length

Reply:
Good observation but also ts lint says 

```
Line breaks are not allowed in import declaration (import-spacing)
```

So I need to ignore a rule anyway